libBuild.sh

```diff
@@ -4,15 +4,26 @@ set -Eeuo pipefail
 
 # Regions that support arm64 architecture
 REGIONS_ARM=(
+  af-south-1
```

Comment:
you're missing `ap-east-1` Asia Pacific (Hong Kong)

Reply:
@bizob2828 NR hasn't published layers to Hong Kong.

Reply:
ah ok. i was just comparing against the blog post from AWS

Reply:
OK, so we do list ap-east-1 in x86 regions, but we don't seem actually to publish to it.